### PR TITLE
ci(e2e): refactor eoa keys

### PR DESCRIPTION
**Pull request by corverroos**
Introduces GCP based EOA keys:
 - Add a eoa key to monitor service
 - Use it to call `omniAVM.SyncToOmni` once per day.
 - Remove relayer-key flag from e2e app and from netman.
 - Move it and monitor key to `eoa` package.
 - Use GCP based keys (or anvil key for devnet)
 - Add eoa keys to devnet and staging genesis.
 - Fund eoa keys for anvil.
 - Remove deployInfo froms Deploy results
 - Move it to app.Definition, where it uses deterministic netman and avs contract addrs.

task: none

### CODE REVIEW

File: e2e/app/avs.go [hunk 1]
```diff
@@ -3,17 +3,13 @@ package app
 import (
```
Comment (corverroos) [inline e2e/app/avs.go:3]: simplified this file

File: e2e/app/definition.go [hunk 2]
```diff
@@ -34,11 +35,10 @@ type DefinitionConfig struct {
 	InfraProvider string
 
 	// Secrets (not required for devnet)
-	DeployKeyFile  string
-	RelayerKeyFile string
```
Comment (corverroos) [inline e2e/app/definition.go:38]: removed this flag

File: e2e/app/definition.go [hunk 3]
```diff
@@ -83,6 +83,29 @@ func (d Definition) Netman() netman.Manager {
 	return d.lazyNetwork.MustNetman()
 }
 
+// DeployInfos returns the deploy information of the OmniPortal and OmniAVS contracts.
+func (d Definition) DeployInfos() types.DeployInfos {
```
Comment (corverroos) [inline e2e/app/definition.go:87]: calculate deployInfos from netman and wellknown avs addresses

**Review by kevinhalliday — APPROVED** [review]

File: e2e/app/definition.go (hunk 2 above)
Comment (pavelnikolov) [reply e2e/app/definition.go:38]: We need to remove this from the ops repo as well.

File: e2e/types/testnet.go [hunk 4]
```diff
@@ -87,13 +86,12 @@ func (t Testnet) HasOmniEVM() bool {
...
 // EVMChain represents a EVM chain in a omni network.
 type EVMChain struct {
-	Name               string // Chain Nam.
-	ID                 uint64 // Chain ID
-	IsPublic           bool
-	IsAVSTarget        bool
-	BlockPeriod        time.Duration
-	FinalizationStrat  netconf.FinalizationStrat
-	AVSContractAddress common.Address
+	Name              string // Chain Nam.
```
Comment (pavelnikolov) [inline e2e/types/testnet.go:89]: nit - was the `.` at the end of `Name` a typo?

**Review by pavelnikolov — APPROVED** [review]

File: e2e/app/definition.go (hunk 2 above)
Comment (corverroos) [reply e2e/app/definition.go:38]: indeed

**Pull request merged**